compose/php.yml

```
@@ -659,6 +659,57 @@ services:
     networks:
       - totara
 
+
+  php-8.4:
+    image: totara/docker-dev-php84
+    container_name: totara_php83
```

This needs to be updated as otherwise theres naming conflict errors when upping the container
```suggestion
    container_name: totara_php84
```

---

Good spotting, updated